app/assets/stylesheets/general/_general.scss

```diff
@@ -123,3 +123,8 @@ strong {
   width: 100%;
   height: auto;
 }
+
+::selection {
+  color:white;
+  background-color: var(--primary);
```

Comment:
I don't think it's wise to use the primary color, as the color might be lost. For example `<mark>` is makes the text colored primary, and buttons have it as their background.

Reply:
Well, the color is set to white so i dont see the conflict with highlighted texts, on the other hand despite only having 3 buttons with primary colored background, it s an issue, yes.